CHANGELOG-developer.next.asciidoc

```diff
@@ -211,12 +211,13 @@ The list below covers the major changes between 7.0.0-rc2 and main only.
 - Add a configuration option for TCP/UDP network type. {issue}40407[40407] {pull}40623[40623]
 - Added debug logging to parquet reader in x-pack/libbeat/reader. {pull}40651[40651]
 - Added filebeat debug histograms for s3 object size and events per processed s3 object. {pull}40775[40775]
-- Simplified GCS input state checkpoint calculation logic. {issue}40878[40878] {pull}40937[40937] 
+- Simplified GCS input state checkpoint calculation logic. {issue}40878[40878] {pull}40937[40937]
```

Comment:
This is not supposed to be here.

Reply:
done in https://github.com/elastic/beats/pull/42914/commits/9cfe7de7438024cda539b7c25a965118f41bcece